### PR TITLE
Autoprefixer

### PR DESCRIPTION
**Pull request by spliner**
Hey, I've implemented basic autoprefixer support. What do you think?

### CODE REVIEW

File: bin/autoless [hunk 1]
```diff
@@ -14,6 +14,7 @@ program
   .option('--interval <ms>', 'How often files are checked for changes', 100)
   .option('--no-watch', "Compile what needs to be compiled and exit")
   .option('--source-map', "Generate source map files next to css files")
+  .option('--auto-prefix <browsers>', "String containing the list of browsers, separated by comma")
```
Comment (jgonera) [inline bin/autoless]: Could you add an example of such a string? What syntax does it have? Should it contain popularity thresholds for browsers?

File: lib/lessfile.js [hunk 2]
```diff
@@ -16,6 +17,14 @@ LessFile.prototype = {
     });
   },
 
+  _getProcessor: function(autoPrefix) {
```
Comment (jgonera) [inline lib/lessfile.js]: Let's name it `_getAutoprefixerProcessor`.

File: package.json [hunk 3]
```diff
@@ -11,7 +11,8 @@
     "watch": "0.9.0",
     "growl": "1.7.0",
     "commander": "2.1.0",
-    "less": "1.7.5"
+    "less": "1.7.5",
+    "autoprefixer-core": "^5.1.7"
```
Comment (jgonera) [inline package.json]: Are we sure that the API won't change until the next major version?

**Comment by jgonera** [comment]
Looks good, but please add tests to `lessfile_test.js`. I would also standardize on `autoprefix` vs `autoPrefix` with the preference for the former since that's the name of the official package.

File: bin/autoless (hunk 1 above)
Comment (spliner) [inline bin/autoless]: These strings are [Browserslist](https://github.com/ai/browserslist) queries. Something like 'last 2 versions', for example.

File: lib/lessfile.js (hunk 2 above)
Comment (spliner) [inline lib/lessfile.js]: Will do.

File: package.json (hunk 3 above)
Comment (spliner) [inline package.json]: You're right about that. I'll take a look and make changes as needed.

**Comment by spliner** [comment]
Got it.

File: bin/autoless (hunk 1 above)
Comment (jgonera) [inline bin/autoless]: OK, let's add an example there, so that it's clear for users.

**Comment by spliner** [comment]
How does it look now?

**Pull request merged**